main.tex

```diff
@@ -132,6 +216,61 @@
 \newpage
 \tableofcontents
 \newpage
+\listoffigures
+\newpage
+\listoftables
+\newpage
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                 Glossary & List of Symbols                    %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\section*{List Of Symbols}
+
+\begin{description}
+
+\item[\parbox{1.25 in}{$\delta$}] Range Modifier
+\item[\parbox{1.25 in}{$\gamma$}] Hit Point Modifier 
+\item[\parbox{1.25 in}{$\sigma$}] Linear Modifier
+\item[\parbox{1.25 in}{$X$}] Enemy Resilience 
+\item[\parbox{1.25 in}{$Y$}] Enemy Damage
+\item[\parbox{1.25 in}{$HP$}] Health Points
+\item[\parbox{1.25 in}{$ST$}] Stamina Points
+\item[\parbox{1.25 in}{$c$}] Current Level
+\item[\parbox{1.25 in}{$L$}] Total Number Of Levels
+\item[\parbox{1.25 in}{$N$}] Number Of Agents  
+\item[\parbox{1.25 in}{$\mu$}] Threshold Percentage 
+\item[\parbox{1.25 in}{$M$}] Number Of Agents Required To Win game
+\item[\parbox{1.25 in}{$N_f$}] Number Of Fighting Agents 
+\item[\parbox{1.25 in}{$N_c$}] Number Of Cowering Agents 
+\item[\parbox{1.25 in}{$HP_{pool}$}] Health pool
+\item[\parbox{1.25 in}{$HP_c$}] Cowering Health Regeneration
+\item[\parbox{1.25 in}{$ST_c$}] Cowering Stamina Regeneration
+\item[\parbox{1.25 in}{$ST_{cower}$}] Cower Stamina Regeneration
+\item[\parbox{1.25 in}{$HP_{cower}$}] Cower Health Regeneration   
+\item[\parbox{1.25 in}{$A_s$}] Attack Base 
+\item[\parbox{1.25 in}{$A_b$}] Attack Bonus
+\item[\parbox{1.25 in}{$A$}] Attack Total  
+\item[\parbox{1.25 in}{$D_s$}] Defense Base   
+\item[\parbox{1.25 in}{$D_b$}] Defense Bonus
+\item[\parbox{1.25 in}{$D$}] Defense Total
+\item[\parbox{1.25 in}{$P_hp$}] Health Potion Value
+\item[\parbox{1.25 in}{$P_st$}] Stealth Potion Value 
+\item[\parbox{1.25 in}{$P$}] Potion Drop Quantity Percentage
+\item[\parbox{1.25 in}{$E$}] Weapon Drop Quantity Percentage
+\item[\parbox{1.25 in}{$N_p$}] Number Of Potions Dropped
+\item[\parbox{1.25 in}{$N_e$}] Number Of Weapons Dropped
+\item[\parbox{1.25 in}{$N_hp$}] Number Of Health Potions Dropped
+\item[\parbox{1.25 in}{$N_st$}] Number Of Stamina Potions Dropped
+\item[\parbox{1.25 in}{$N_a$}] Number Of Swords Dropped
+\item[\parbox{1.25 in}{$N_d$}] Number Of Shields Dropped
```

Comment:
Built pdf currently leaves a lot of empty space since list overruns to next page. Not a big issue though

Reply:
really? I built it in a blank overleaf life and it worked fine :/

Reply:
oh fair, we can probably figure out formatting stuff like this later anyways